### ISSUE TITLE
Stuck on "Sending" AWS SES & SNS

### ISSUE THREAD

**Issue by Sepremex** [mettle/sendportal#104]
I have setup the SES and SNS, I did a few test before the final run, I also request the AWS SES team to put the account on production which was approved...
I'm sending 14 segments each with about 10K emails making a total of 120K(more or less) emails, so far it has send only about 90 emails, all I can see is that is stuck in "Sending" Cron job" is working, AWS doesn't report any error, on the website doesn't show any errors either, I check the logs on laravel and the server it self but no errors or warning, what could be the problem here?
This has 72hr until now since we send the emails, but like I said it is stuck on "Sending" it did sent a few for that is it... where else can look to see whats going on... 
Thanks...

**Comment by JonoB** [comment]
@Sepremex - can you confirm which queue driver you are using.

See https://sendportal.io/docs/getting-started/configuration-and-setup#additional-configuration for more information.

**Comment by Sepremex** [comment]
> @Sepremex - can you confirm which queue driver you are using.
> 
> See https://sendportal.io/docs/getting-started/configuration-and-setup#additional-configuration for more information.

I'm using this on the .env
CACHE_DRIVER=file
QUEUE_CONNECTION=sync
Should I use DataBase?

**Comment by Sepremex** [comment]
I changed it to "database" but in realty not a single mail was sent, I also change the status in the database to 2, then it went to 4 after the cronjob ran, but didn't sent any email at all...
plus, using sync I find out that is requiring more memory over 4GB on ram... could this be the problem?

**Comment by Sepremex** [comment]
I think I got it fixed, it was more that just a simple task, I had to request a server access to see more info about the resources and configs, there was a whole chain of errors on the server.
Redis was installed but was not active, Horizon also was installed but not active for some reason, the worst part is that there was no errors at all regarding those services when a request or cache was made... 
So I did some debuggin on the server, build a basic socket to test redis, I also installed supervisord for the horizon and other apps, went back to SendPortal change it from database to redis, and BOOMM!!!! 400 emails in less than a minute not even AWS SES saw that coming! lol... that was last night..

Today we just sent out 300K emails 45 segments, is going to take 3 days or so due to AWS SES limitations, but is working, Horizon is showing some data, redis is working fine and SendPortal is showing results!

Thank you!
